pytest_mh/utils/journald.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
+
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+
+from .. import MultihostHost, MultihostUtility
+from ..ssh import SSHProcessResult
+
+__all__ = ["JournaldUtils"]
+
+
+class JournaldUtils(MultihostUtility):
+    """
+    Perform journald related tasks.
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        """
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+
+    def setup(self) -> None:
+        """
+        Called before execution of each test.
+        """
+        self.clear()
+
+    def clear(self) -> None:
+        """
+        Reset timestamp
+        """
+        self.test_start = self.host.ssh.exec(["date", "--rfc-3339=ns"]).stdout[0:19]
```

Comment:
If you want to clear it, then you should use `--vacuum` 

But if this is the intention, then I'm fine with it to unblock your work. But I'd like to rework it, if possible, to "archive existing journal", "vacuum between tests", "restore old journal" to properly clean up after test run and instead of relying on timestamp. But this does not have to be part of this PR.